src/ResourceTemplateValidator.js

```diff
@@ -120,7 +120,7 @@ const validateTemplateRefsExist = async (resourceTemplate) => {
     const refs = propertyTemplate.valueConstraint?.valueTemplateRefs || []
     await Promise.all(refs.map(async (resourceTemplateIdRef) => {
       const found = await foundResourceTemplate(resourceTemplateIdRef)
-      if (!found) {
+      if (!found && resourceTemplateIdRef) { // ignore blank referenced templates elements
```

Comment:
For clarity, please consider using lodash: `! _.isEmpty(resourceTemplateIdRef)`.